content/docs/04-clusters/06-cluster-management/05-backup-restore.md

```diff
@@ -11,320 +11,493 @@ import WarningBox from 'shared/components/WarningBox';
 import InfoBox from 'shared/components/InfoBox';
 import PointsOfInterest from 'shared/components/common/PointOfInterest';
 
-# Overview
 
-Palette provides two ways to back up and restore Kubernetes clusters:
+# Cluster Backup and Restore
 
-* Cluster Backup and Restore for a single cluster which is managed from within the cluster.
-* [Workspace](/workspace/workload-features#workspaceoperator) Backup and Restore for multiple clusters managed from workspaces.
+A backup is a persistent state of Kubernetes clusters, including the native objects such as Pods, DaemonSets, Services, and persistent volumes. A backup allows you to save the current state of a cluster and restore it at a later point in time if needed. You can restore a backup to the same or a different cluster. Internally, Palette leverages an open-source tool called *Velero* to provide backup and restore capabilities. 
 
-# Cluster Backup and Restore
 
-Palette provides a convenient backup option to back up the Kubernetes cluster state into object storage and restores it at a later point in time if required to the same or a different cluster. Besides backing up Kubernetes native objects such as Pods, DaemonSets, and Services, persistent volumes can also be snapshotted and maintained as part of the Backup. Internally, Palette leverages an open-source tool called Velero to provide these capabilities. In addition, multiple backups of a cluster can be maintained simultaneously.
+Palette allows you to schedule and take a backup of a specific cluster or an entire [workspace](/workspace). Also, you can maintain multiple backups of a cluster or a workspace. This guide provides the instructions for creating and restoring a cluster backup. You can refer to the [Manage Palette Workspace](/workspace/workload-features#managepaletteworkspace) guide to create and restore a workspace backup. 
+
+
+Before you create a backup, the initial step is to configure a backup location in Palette. A backup location is an object storage in the cloud. Palette supports the following object storages as the backup location.
```

Comment:
```suggestion
Before you create a backup, the initial step is to configure a backup location in Palette. A backup location is an object storage. Palette supports the following object storages as the backup location.
```
It doesn't have to be in the cloud, you can deploy MinIO to an on-prem data center. Be careful about assuming something is only for the cloud. We deal with both on-prem and cloud, so we have to be careful when stating what environment something is available in.